lib/counter_culture.rb

```diff
@@ -13,7 +13,13 @@ module ActiveRecord
 
     module ClassMethods
       # this holds all configuration data
-      attr_reader :after_commit_counter_cache
+      def after_commit_counter_cache
+        config = @after_commit_counter_cache
+        if !config && superclass.respond_to?(:after_commit_counter_cache)
+          config = superclass.after_commit_counter_cache
+        end
+        config
+      end
```

Comment:
It seems to me like you'd want to preserve both, ideally. For example, if I were to create a `TwitterReview`, I would expect both the `reviews_count` and `twitter_reviews_count` on a product to go up. Can you just merge them up front inside of `.counter_culture`? Something like:

```ruby
def counter_culture(relation, options = {})
  unless @after_commit_counter_cache
    if superclass.respond_to?(:after_commit_counter_cache) && superclass.after_commit_counter_cache
      @after_commit_counter_cache = superclass.after_commit_counter_cache
    else
      # what was here previously
    end
  end
end
```